src/HIE/Bios/Cradle.hs

```diff
@@ -505,12 +503,26 @@ findFile p dir = do
 -- | Call a process with the given arguments and the given stdin
 -- in the given working directory.
 readProcessWithExitCodeInDirectory
```

Comment:
Semantics of this function have changed. Originally, it was just a wrapper that set the working dir, now it does a bit more. 
So, adapt documentation.

Reply:
Done.